.github/CODEOWNERS

```
@@ -1,9 +1,47 @@
-# Require legal approval for all new graphics
-android/app/src/main/res/drawable*/ @organicmaps/legal
-android/app/src/main/res/fonts/ @organicmaps/legal
-android/app/src/main/res/mipmap*/ @organicmaps/legal
-data/*.ttf @organicmaps/legal
-data/resources*/ @organicmaps/legal
-data/search-icons/ @organicmaps/legal
-data/styles/clear/style-*/ @organicmaps/legal
-iphone/Maps/Images.xcassets/ @organicmaps/legal
+# Design
+android/app/src/main/res/drawable*/ @organicmaps/design
+android/app/src/main/res/fonts/ @organicmaps/design
+android/app/src/main/res/mipmap*/ @organicmaps/design
+data/*.ttf @organicmaps/design
+data/resources*/ @organicmaps/design
+data/search-icons/ @organicmaps/design
+data/styles/clear/style-*/ @organicmaps/design
```

```suggestion
data/styles/default/light/*/*.png @organicmaps/design
data/styles/default/light/*/*.svg @organicmaps/design
data/styles/default/dark/*/*.png @organicmaps/design
data/styles/default/dark/*/*.svg @organicmaps/design
```

Will the `data/styles/default/light/*/*.png` notation work to match e.g. `data/styles/default/light/hdpi/ferry.png`?

---

Probably `data/styles/default/light/**/*.svg`, but I haven't tested.

---

Added